### ISSUE TITLE
Simple A-Z list of transects

### ISSUE THREAD

**Issue by IanMiddlebrook** [BiologicalRecordsCentre/UKBMS-online#211]
Hi @DavidRoy 

On the old website there was an option to look for transect sites by name in an A-Z list.

Whilst we have a great new map to help locate transects, a simple list would also be useful (if you already know the name of the site, but take a while to find it on thee map). I've had a request for this today.

Would this be possible at some point?

Thanks, Ian

**Comment by DavidRoy** [comment]
Good idea.  Rich - could you create this, adapting the report used to create the map
https://ukbms.org/sites

**Comment by burkmarr** [comment]
I've scheduled some time to look at this early next week.
